packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/overview/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Overview
+---
+
+Most Okta API endpoints require that you include an API token with your request. Currently, this API token takes the form of an SSWS token that you generate in the Admin Console. With OAuth for Okta, you are able to interact with Okta APIs using scoped OAuth 2.0 access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
+
+## Prerequisites
+
+To use this guide, you need the following:
+
+* An Okta Developer Edition Org. (Don't have one? [Create an org for free](https://developer.okta.com/signup).)
+* [Postman client](https://www.getpostman.com/downloads/) to test requests with the access token. See [Get Started with the Okta APIs](https://developer.okta.com/code/rest/) for information on setting up Postman.
+
+## Use the Client Credentials grant flow
+
+For machine-to-machine use cases where a backend service or a daemon has to call Okta APIs, use the Client Credentials grant flow with an OAuth service app. The Client Credentials grant flow is the only grant flow supported with the OAuth service app when you want to mint access tokens that contain Okta scopes.
+
+The following are the high-level steps required to perform the Client Credentials grant flow with an OAuth service app:
+
+1. Create a public/private JWKS keypair and extract the public key to pass it along with the client creation API call.
```

Comment:
Any reason all of these numbered statements have same number `1.`? Some markdown magic?

Reply:
yes, you don't have to consecutively number in markdown...it does it automagically for you.